CMakeLists.txt

```diff
@@ -281,6 +309,8 @@ endif ()
 option(USE_OPENSSL "Enable use of OpenSSL if available" ON)
 option(SEND_TELEMETRY_DEFAULT "The default value for whether to send telemetry" ON)
 option(REGRESS_CHECKS "PostgreSQL regress checks through installcheck" ON)
+option(ENABLE_WARNINGS_AS_ERRORS
```

Comment:
I'm actually with Sven here, would prefer the default be OFF. But, I don't feel strongly and won't block the PR because of this. Just my 2 cents.